worker/lease/manager.go

```diff
@@ -175,8 +175,13 @@ func (manager *Manager) choose(blocks blocks) error {
 	case check := <-manager.checks:
 		return manager.handleCheck(check)
 	case manager.now = <-manager.nextTick(manager.now):
```

Comment:
Doesn't nextTick still copy Leases?
Is there a way to not do this?

Reply:
Yeah, that's coming in the next PR.